tests/testthat/_snaps/slice-chop.md

```diff
@@ -1,3 +1,82 @@
+# `indices` are validated
+
+    Code
+      vec_chop(1, indices = 1)
+    Condition
+      Error:
+      ! `indices` must be a list of index values, or `NULL`.
+
+---
+
+    Code
+      (expect_error(vec_chop(1, indices = list(1.5)), class = "vctrs_error_subscript_type")
+      )
+    Output
+      <error/vctrs_error_subscript_type>
+      Error:
+      ! Can't subset elements.
+      x Can't convert from <double> to <integer> due to loss of precision.
```

Comment:
The errors of `vec_chop()` aren't perfect, but that is partially due to not having exposed `error_call` and `error_arg` like `list_unchop()`. But that is big enough to be its own PR. I have moved a few tests towards snapshot tests rather than just `expect_error()` calls so we will eventually see the improvements when we add those arguments.